cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/JmsSource.java

```diff
@@ -328,7 +330,7 @@ public JmsProvider getJmsProvider() {
 
     public JmsPluginConfig() {
       messagesToReceive = 50;
-      connectionFactoryName = "ConnectionFactory";
+      connectionFactoryName = CONNECTION_FACTORY;
     }
 
     public JmsPluginConfig(String destinationName, @Nullable Integer messagesToReceive, String initialContextFactory,
```

Comment:
Actually this is already there and not part of the PR, I could send another one to change it.